.travis.yml

```diff
@@ -4,50 +4,65 @@ cache: ccache
 
 matrix:
   include:
-  #Ubuntu
-  - name: Ubuntu (Debug)
+  #Ubuntu Bionic (on docker)
+  - name: Ubuntu Bionic (Debug)
     if: tag IS NOT present
-    os: linux
-    dist: xenial
-    group: stable
-    env: BUILDTYPE=Debug
-  - name: Ubuntu (Release)
+    services: docker
+    before_install: docker build -t img .ci
+    script: docker run --mount "type=bind,source=$(pwd),target=/src" -w="/src" img
+          bash .ci/travis-compile.sh --server "Debug"
+  - name: Ubuntu Bionic (Debug)
```

Comment:
I guess this is a typo, should be `(Release)`

Reply:
Yes it is, I had to copy all the configs around, this is why we hate WET!